ui/web/src/main/java/com/ttbmp/cinehub/ui/web/buyticket/ChooseSeatsViewController.java

```diff
@@ -0,0 +1,99 @@
+package com.ttbmp.cinehub.ui.web.buyticket;
+
+import com.ttbmp.cinehub.app.dto.ProjectionDto;
+import com.ttbmp.cinehub.app.dto.TicketDto;
+import com.ttbmp.cinehub.app.usecase.buyticket.BuyTicketUseCase;
+import com.ttbmp.cinehub.app.usecase.buyticket.Handler;
+import com.ttbmp.cinehub.app.usecase.buyticket.request.GetNumberOfSeatsRequest;
+import com.ttbmp.cinehub.app.usecase.buyticket.request.GetProjectionRequest;
+import com.ttbmp.cinehub.ui.web.domain.Projection;
+import com.ttbmp.cinehub.ui.web.domain.Seat;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Palmieri Ivan
+ */
+@Controller
+public class ChooseSeatsViewController {
+
+    private final String projectionList="projectionList";
+
+    @PostMapping("/choose_seat")
+    public String chooseSeats(@ModelAttribute("projection") Projection projection,
+            Model model) {
+
+        BuyTicketUseCase buyTicketUseCase = new Handler(new BuyTicketPresenterWeb(model));
+        buyTicketUseCase.getProjectionList(new GetProjectionRequest(
+                projection.getMovieId(),
+                projection.getCinemaId(),
+                LocalDate.parse(projection.getDate()),
+                projection.getStartTime(),
+                projection.getHallId()
+        ));
+        buyTicketUseCase.getListOfSeat(new GetNumberOfSeatsRequest(((ArrayList<ProjectionDto>)model.getAttribute(projectionList)).get(0))); //Mi ritorna sempre una lista
+        model.addAttribute("boolean1", false);
```

Comment:
You add into the model a lot of unutilized constant value as attribute, why don't you just add them in the HTML?

Reply:
Only in the case of those three Booleans I had to resort to that system. Since they are not displayed correctly when created from HTML